content/actions/managing-issues-and-pull-requests/scheduling-issue-creation.md

```diff
@@ -47,12 +43,31 @@ In the tutorial, you will first make a workflow file that uses the [`imjohnbo/is
           issues: write
         steps:
           - name: Create team sync issue
-            uses: imjohnbo/issue-bot@3d96848fb5e9a4a473bb81ae62b4b4866a56e93a
-            with:
-              assignees: "monalisa, doctocat, hubot"
-              labels: "weekly sync, docs-team"
-              title: "Team sync"
-              body: |
+            run: |
+              new_issue_url=$(gh issue create \
+                --repo "$REPO" \
+                --title "$TITLE" \
+                --assignee "$ASSIGNEES" \
+                --label "$LABELS" \
+                --body "$BODY")
+              if [[ $PINNED == true ]]; then
+                gh issue pin "$new_issue_url" --repo "$REPO"
+              fi
+              if [[ $CLOSE_PREVIOUS == true ]]; then
+                previous_issue_number=$(gh issue list \
+                  --repo "$REPO" \
+                  --label "$LABELS" \
+                  --json number \
+                  --jq .[1].number)
```

Comment:
It looks like this gives an error if quotes are missing. I think we should also use `0` instead of `1` to get the first item from the list.

```suggestion
                  --jq '.[0].number')
```